app/controllers/field_slips_controller/index.rb

```diff
@@ -0,0 +1,80 @@
+# frozen_string_literal: true
+
+# see field_slips_controller.rb
+module FieldSlipsController::Index
+  private
+
+  # index subactions:
+  # methods called by #index via the dispatch table in FieldSlipsController
+
+  # checked by ApplicationController#index
+  def default_index_subaction
+    list_all
+  end
+
+  def list_all
+    query = create_query(:FieldSlip, :all, by: :created_at)
+    show_selected_field_slips(query)
+  end
+
+  # This does not appear to get used by any of the new functionality
+  # so I think this is dead code.
+  #
+  # # Displays matrix of selected FieldSlips (based on current Query).
+  # def index_query_results
+  #   query = find_or_create_query(:FieldSlip, by: params[:by])
+  #   show_selected_field_slips(
+  #     query, id: params[:id].to_s, always_index: true
+  #   )
+  # end
```

Comment:
@mo-nathan This is the code that puts you back in the index _at the page containing the last field slip you looked at_, so it may be expected behavior. 

Say you're checking out a field slip on the third page of a filtered index, then you step through the "previous" field slips for a few dozen in the "show" view, then you go back to the index. This method (with the `:id` param) is like a pointer that tells the index which page in the results you would expect to be.